book/website/foreward/history.md

```diff
@@ -0,0 +1,57 @@
+(fw-history)=
+
+# History of *The Turing Way* project
+
+## Founding (2018 - 2019)
+
+*The Turing Way* is the flagship project of the Tools, Practices, and Systems programme.
+Founded by Dr Kirstie Whitaker, the project began as a Book of Reproducible Research in 2019.
+Dr. Whitaker brought together a team of [allies and contributors](https://github.com/the-turing-way/the-turing-way/blob/main/ways_of_working.md#previous-project-members), who were already championing reproducible practices in data science, to co-develop the first version of **The Turing Way** book.
+The project team included researchers, librarians, data scientists and Research Engineers, from both The Alan Turing Turing and the institutes across the UK.
+The project has its roots in the [Mozilla Open Leadership](https://mozilla.github.io/leadership-training/), a programme that led to Kirstie's Mozilla Fellowship in 2016, which influenced her as a researcher and strengthened her commitment to working openly.
+
+*The Turing Way* has been funded by The Alan Turing Institute since the very beginning.
+Due to close relationships with the Jupyter Project, the book was written in JupyterBook.
+A GitHub repository was made for the project, and internal processes decided in how to acknowledge authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (such as the Collaboration Cafe and Book Dash).
```

Comment:
```suggestion
A GitHub repository was made for the project, and internal processes decided how to acknowledge authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (such as the Collaboration Cafe and Book Dash).
```
We can point specifically to the Authorship chapter in the community handbook here - we should probably also be clearer on the "internal processes", was it Kirstie or the original project team who made the authorship decisions?

Reply:
Just to flag - the use of "i.e." here is causing CI to fail

Reply:
Edited to "for example"